### ISSUE TITLE
KeyError: 'imaging_quality_preprocessing_mode'

### ISSUE THREAD

**Issue by quantumiracle** [Vchitect/VBench#35]
Hi, there is an error using python script for evaluation:

```
python3.10/site-packages/vbench/imaging_quality.py", line 30, in technical_quality
    preprocess_mode = kwargs['imaging_quality_preprocessing_mode']
KeyError: 'imaging_quality_preprocessing_mode'
```

**Comment by NattapolChan** [comment]
Can you check if vbench package you installed and the python script are from the same version. I cannot replicate the issue with the latest version.

**Comment by ziqihuangg** [comment]
Hi, I'm closing the issue as there's no follow-up question, but feel free to re-open it anytime if you have any further questions!

**Comment by lmxyy** [comment]
@quantumiracle You need to pass the `imaging_quality_preprocessing_mode` into kwargs as in [evaluate.py](https://github.com/Vchitect/VBench/blob/58642fd51e272b93b7a132e19ede3a82a1d69ab9/evaluate.py#L143) when calling `VBench.evaluate`.

@NattapolChan @ziqihuangg In your [README](https://github.com/Vchitect/VBench?tab=readme-ov-file#python) python example, you also do not specify this argument. I guess that was why we met this error and you could reproduce it.

**Comment by NattapolChan** [comment]
```python
time CUDA_VISIBLE_DEVICES=0 python3 evaluate.py --dimension imaging_quality \
  --mode=custom_input \
  --videos_path './sampled_videos' \
  --output_path test_${date_str}_imaging_quality/
```
```
args: Namespace(output_path='test_2024-07-27_imaging_quality/', full_json_dir='/home/users/nattapol/VBench/vbench/VBench_full_info.json', videos_path='./sampled_videos', dimension=['imaging_quality'], load_ckpt_from_local=None, read_frame=None, mode='custom_input', custom_input=False, prompt='', prompt_file=None, category=None, imaging_quality_preprocessing_mode='longer')
start evaluation
Evaluation meta data saved to test_2024-07-27_imaging_quality/results_2024-07-27-18:12:09_full_info.json
cur_full_info_path: test_2024-07-27_imaging_quality/results_2024-07-27-18:12:09_full_info.json
^M  0%|          | 0/1 [00:00<?, ?it/s]
^M100%|██████████| 1/1 [00:34<00:00, 34.39s/it]
^M100%|██████████| 1/1 [00:34<00:00, 34.39s/it]
Evaluation results saved to test_2024-07-27_imaging_quality/results_2024-07-27-18:12:09_eval_results.json
done

real    2m9.035s
user    1m43.805s
sys     0m16.577s
```

It does work fine for me. The argument should be set by default to 'longer'. It should work fine even without `imaging_quality_preprocessing_mode` flag. I think it might be due to version conflict between evaluate.py and pypi package. Can you provide the commit hash of vbench to reproduce this?

**Comment by NattapolChan** [comment]
Thanks for pointing this out. I didn't check calling VBench.evaluate() directly. This should be fixed in [2048fb0](https://github.com/Vchitect/VBench/commit/2048fb0e9e0b158e99bb59d525fd19a16627ef8c).
